.changeset/tough-dolphins-kick.md

```diff
@@ -0,0 +1,6 @@
+---
+"@sveltejs/adapter-cloudflare-workers": minor
+"@sveltejs/adapter-cloudflare": minor
+---
+
+support platform emulation configuration via the `platformProxy` adapter option
```

Comment:
```suggestion
feat: support platform emulation configuration via the `platformProxy` adapter option
```

Reply:
Thanks for the feedback. Amended to previous commit to keep it clean.